clients/fides-js/src/lib/preferences.ts

```diff
@@ -59,13 +67,21 @@ export const updateConsentPreferences = ({
     });
   });
 
-  // 1. DEFER: Save preferences to Fides API
+  // 1. Save preferences to Fides API
   debugLog(debug, "Saving preferences to Fides API");
-  const privacyPreferenceCreate: PrivacyPreferencesCreateWithCode = {
+  const privacyPreferenceCreate: PrivacyPreferencesRequest = {
     browser_identity: cookie.identity,
     preferences: fidesUserPreferences,
+    privacy_experience_history_id: experienceHistoryId,
+    user_geography: userLocationString,
+    method: consentMethod,
   };
-  patchUserPreferenceToFidesServer(privacyPreferenceCreate, debug);
+  patchUserPreferenceToFidesServer(
```

Comment:
What happens if this fails - like a 500 error?

We should still continue and save the cookie and update the object locally in that case so that everything works for the user.

Reply:
good point, I'll update

Reply:
This method will always resolve, even if we don't have a 200 resp:

```
if (!response.ok) {
    debugLog(
      debug,
      "Error patching user preference Fides API. Response:",
      response
    );
  }
  return Promise.resolve();
```